presenter/Presenter.py

```diff
@@ -2,20 +2,21 @@
 import pandas as pd
 import requests
 
+from presenter.GraphSynthesizer import compute_presented_name
 from presenter.consts import BOOKS_PRESENTED, API_ENDPOINT
 
 data = requests.get(url=f"{API_ENDPOINT}/api/books?presented_amount={BOOKS_PRESENTED}").json()
 
 df = pd.DataFrame(data)
 
-plt.bar(df['book_id'], df['avg_score'], color='skyblue')
+plt.bar(compute_presented_name(df), df['avg_score'], color='skyblue')
 
-plt.xlabel('book_id')
+plt.xlabel('book')
 plt.ylabel('avg_score')
 
 plt.title(f'Top {BOOKS_PRESENTED} Books by Rating')
 
 plt.xticks(rotation=45)
 
 plt.tight_layout()
-plt.show()
\ No newline at end of file
+plt.show()
```

Comment:
In general, don't write code in the outer scope. Always put code inside a function or a scope of some sort.

Reply:
I agree, will be extracted now.